Mapify/Patches/GlobalShopControllerPatch.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Linq;
+using DV.Shops;
+using HarmonyLib;
+
+namespace Mapify.Patches
+{
+    [HarmonyPatch(typeof(GlobalShopController), nameof(GlobalShopController.InitializeShopData))]
```

Comment:
Do we need to be doing this for everyone every time? I assume you use this to know what items to add to the enum, so it should probably be behind an `#if DEBUG` directive or something similar.

Reply:
I'll add a `#if DEBUG`.